chipsec/cfg/8086/glk.xml

```diff
@@ -9,7 +9,7 @@
   <!-- Information                          -->
   <!--                                      -->
   <!-- #################################### -->
-  <info family="atom" detection_value="706ax">
+  <info family="atom" detection_value="0x7006a"> <!-- verify detection val -->

```

Comment:
Hello @dscott90 , this change seems to have reverted what https://github.com/chipsec/chipsec/pull/1735 ("Fix CPUID detection value of Gemini Lake (GLK) ") did: it replaced `7006a` with `706ax`, providing example of values.

Was there something wrong with using `0x706a0-0x706af` as the new `detection_value` for GLK?

Reply:
There was a conflict with what we were merging in and we made a note to verify. Looks like we chose the wrong one. Thanks for pointing it out. I just opened a PR #1910  to re-fix it.

Reply:
Thanks for your quick reply! The PR looks good to me.